File: C99/1000-1099/1040-1049/1048.c
#include <stdio.h>

int main()
{
    double n, r;
    int p;
    scanf("%lf", &n);
    if (n >= 0 && n <= 400)
    {
        r = n * 0.15;
        n += r;
        p = 15;
    }
    else if (n > 400 && n <= 800)
    {
        r = n * 0.12;
        n += r;
        p = 12;
    }
    else if (n > 800 && n <= 1200)
    {
        r = n * 0.10;
        n += r;
        p = 10;
    }
    else if (n > 1200 && n <= 2000)
    {
        r = n * 0.07;
        n += r;
        p = 7;
    }
    else if (n > 2000)
    {
        r = n * 0.04;
        n += r;
        p = 4;
    }
    printf("Novo salario: %.2f\n", n);
    printf("Reajuste ganho: %.2f\n", r);
    printf("Em percentual: %d %\n", p);
    return 0;
}

File: python/1000-1099/1040-1049/1048.py
entry = float(input())
salReaj = 0
reajust = 0

if entry <= 400:
    salReaj = entry * (15 / 100)
    reajust = 15
if entry > 400 and entry <= 800:
    salReaj = entry * (12 / 100)
    reajust = 12
if entry > 800 and entry <= 1200:
    salReaj = entry * (10 / 100)
    reajust = 10
if entry > 1200 and entry <= 2000:
    salReaj = entry * (7 / 100)
    reajust = 7
if entry > 2000:
    salReaj = entry * (4 / 100)
    reajust = 4
print(f"Novo salario: {entry + salReaj:.2f}")
print(f"Reajuste ganho: {salReaj:.2f}")
print(f"Em percentual: {reajust} %")